src/style.css

```
@@ -0,0 +1,58 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+a {
+  text-decoration: none;
+  color: #000;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  height: 60px;
+  padding: 0 50px;
+}
+
+.logo {
+  margin-right: auto;
+  width: 40px;
+  height: 40px;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+}
+
+.nav__link {
+  text-transform: uppercase;
+  font-size: 12px;
+  padding-right: 20px;
```

```suggestion
  margin-right: 20px;
```
Let's use outer spacings (not inner)